src/utils/flagging.R

```diff
@@ -22,6 +22,14 @@ box::use(lubridate)
 #' @param first_since `numeric` vector indicating when to flag the first positive
 #'     values. For instance, `180` would indicate flagging whenever there is a
 #'     positive value after 180 days.
+#' @param first_since_minimums `numeric` vector of minimum values such that if `x`
+#'     is above it, no flags are generated. If a single value, the value is
+#'     recycled. If the same length as `first_since`, the thresholds are matched
+#'     one-to-one with `first_since`. Since `first_since` is concerned with the
+#'     first value in a period of time, it only triggers on the first instance.
+#'     The minimum is therefore not just on the single first day, but is measured
+#'     against the following 60 days (which matches the period in which an email
+#'     would be generated, older flags would not generate any email).
```

Comment:
how does 60 match the period in which an email would be generated?

Reply:
Okay, so I think the confusion here and below definitely deserves some clarification. But the whole redesign basically came out of some issues with the emails, mainly unique to IDMC, where delays in reporting can generate older flags.

So, for instance, data could be added for 30 days ago that would generate a flag on the first displacement since a year.

Let's say 400 people in Mali were displaced on October 1. However, maybe there was also displacement last week, 300 people in Mali on October 20. And the data was added today, the 27th of October. 

Previously, the flag would just look at displacement from that first alert. But, that meant the emails messaging would maybe miss reporting on all the displaced values since then. It would say 400 displaced on October 1, ignoring the latter data. So, we wanted to instead message about all of the available data SINCE the flag was generated.

So that was what `rs_next_60` is. Basically, emails are not generated if the alerts are older than 60 days, so with this sum we can assume we will have all latest displacement figures in our message. In this case, 700 people have been displaced SINCE October 1 (and a plot to contextualise all this).

Now, we also got feedback to not flag if displacement is under a certain amount for these first since flags. But, at what level do we look at that? In this case, the threshold is 500, so if we just look at October 1, we wouldn't flag. But if we look across the potential period for flagging, we have 700, so we would flag. So what I did was look across the whole next 60 days and see if that meets that threshold.

So basically we have system that needs to look back a period of time to check that nobody was displaced, and over the next period of time to make sure that a certain amount of people have been displaced. Maybe we could only look backwards, but what we are doing now is looking from the date of alert, when we would list the start date of the alert, and in our messaging, say that displacement started. So there is definitely value there. And hard to know how to easily look back and say in the past 60 days, we see X displacement, but past 180 days + 60 we see none. Might not be that much simpler.

But, putting this in an open issue so we can discuss further!